lib/ansible/modules/packaging/os/dnf.py

```diff
@@ -469,9 +500,10 @@ def main():
             conf_file=dict(default=None, type='path'),
             disable_gpg_check=dict(default=False, type='bool'),
             installroot=dict(default='/', type='path'),
+            autoremove=dict(default=True, type='bool'),
```

Comment:
In fedora 25 when you do `dnf remove jenkins-core` for example, the default behavior is to autoremove the dependencies. Setting this value to `False` will change the expected behavior